backend/docs/plantuml/statusChange.puml

```diff
@@ -18,17 +18,15 @@
     ' status states
     state RECEIVED
     state IN_PROCESSING
-    state HAS_ERRORS
+    state PROCESSED
     state RECEIVED
     state APPROVED_FOR_RELEASE
```

Comment:
While we're at it we could also rename "APPROVED_FOR_RELEASE" to "RELEASED", the former is unnecessarily complicated and wordy.

Reply:
(I support this change at some point, but agnostic as to whether in this PR)